trytond/backend/postgresql/database.py

```diff
@@ -53,6 +53,7 @@
 _minconn = config.getint('database', 'minconn', default=1)
 _maxconn = config.getint('database', 'maxconn', default=64)
 _default_name = config.get('database', 'default_name', default='template1')
+_slow_threshold = config.getfloat('database', 'log_time_threshold', default=-1)
```

Comment:
Je me demande si un default à 0. N'est pas mieux. Tu fais un test `if _slow_threshold` et ainsi si tu mets une valeur de -1 et bien tu times tout.

Reply:
Dans ce cas je vire le logging "standard" de Tryton là-dessus... why not